data/map.txt

```diff
@@ -30694,3 +30700,2835 @@ planet Zug
+	spaceport "	If you stare too long, the nearby guards gesture more aggressively towards you, suggesting you keep your attention to yourself."
+	government "Gegno Vi"
+	"required reputation" 1
+	bribe 0
+	security 0
+
+planet "Oup Je"
+	landscape land/canyon4
+	description "The many beveling formations here provide the Gegno Scin with a rich amount of materials for their cities and ships. Lower down below are small woodlands that seem to be stripped down one way or another, with few living areas scattered about. It is apparent that this planet has been worn down over time, both to natural and unnatural causes."
+	spaceport "The Scin here have different gear than those leaving and entering the spaceport, suggesting they have a particular place on the surface to be, most likely in the mines. They seem quite interested in you compared to other Gegno being that a majority of their days involve seeing mostly materials and machinery."
+	government "Gegno Scin"
+	"required reputation" 1
+	bribe 0
+	security 0
+
+planet "Enn Bue"
+	landscape land/canyon01
+	description "Lengthy oval-like shipyards speckle the surface of this otherwise inhospitable desert planet, home to most of the Gegno Scin's ship production. A majority of wealthy materials come from the neighboring gas giant's moons; however, the strong gravitational pull occasionally generates natural disasters that ravage the surface. Despite all of this, the Scin remain vigilant and efficient in their craftworks."
+	spaceport "Enveloped in a large black dome cap, this spaceport is a lot quieter inside than the loud winds and storms often found outside. It has definitely seen a severe duststorm or two, but still holds tight together. The Gegno Scin here give you lengthy stares, and oddly enough you notice that they take turns to observe you. One will look up then look back down to themselves followed by another's attention."
+	government "Gegno Scin"
+	"required reputation" 1
+	bribe 0
+	security 0
+
+planet Cipi
+	landscape land/mountain5
+	description "Cipi's forestries and mountains are a suitable place for the Gegno Scin's less active population, home to more quiet industries. The population of this planet is limited, and architecture is built on the alpines to avoid hurting the natural landscapes of the planet. This provides the Scin with healthy crop and hunting seasons to supply their main population without the interference of polluting industry."
+	spaceport "Rather than inside the mountain, the spaceport is built at the very edge supported underneath by several pylons in a half-oval formation. The valley down below, between you and the mountain range just a bit further away, seems almost cramped between the formations. Occasionally, a few Halite fighters buzz by at low altitudes and then sharply turn off into the distance, though you expect them to be louder than they really are."
```

Comment:
```suggestion
	spaceport "Rather than inside the mountain, the spaceport is built at the very edge, supported underneath by several pylons in a half-oval formation. The valley down below, between you and the mountain range just a bit further away, seems almost cramped between the formations. Occasionally, a few Halite fighters buzz by at low altitudes and then sharply turn off into the distance, though you expect them to be louder than they really are."
```